src/components/PlanRoom/RoomTables/Types.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react';
```

Comment:
Type definitions should be stored probably in the folder called `model` as it is right now.

Reply:
despite the transfer there is a need for this line of code to be in the file